pallets/asset-manager/src/weights.rs

```diff
@@ -54,18 +54,24 @@ use sp_std::marker::PhantomData;
 pub trait WeightInfo {
 	fn register_asset() -> Weight;
 	fn set_asset_units_per_second() -> Weight;
+	fn change_existing_asset_type() -> Weight;
 }
 
 /// Weights for pallet_asset_manager using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn register_asset() -> Weight {
-		(52_572_000 as Weight)
+		(50_651_000 as Weight)
```

Comment:
Needs proper benchmark. Let me know when you want me to run this.

Reply:
I actually run it on the target machine, but feel free to redo it